apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -1513,6 +1516,7 @@
           "DELETE": "Remove selected",
           "TABLE_RESOURCE_ID": "id",
           "TABLE_RESOURCE_NAME": "Name",
+          "TABLE_VO_NAME": "Virtual Organization",
```

Comment:
please change this to only organization. We are trying in new gui to not use term vo.